tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -405,7 +405,7 @@ public function testConnectionIsClosed()
     {
         $this->_conn->close();
 
-        $this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
+        //$this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
 
         $this->_conn->quoteIdentifier('Bug');
```

Comment:
What happens if the platform cannot be fetched? This code will likely cause a method call on `null`.